models/graphql/shopify_gql__customer_emails.sql

```diff
@@ -0,0 +1,68 @@
+{{ config(enabled=var('shopify_api', 'rest') == var('shopify_api_override','graphql')) }}
+
+with customer_emails as (
+
+    select 
+        {{ dbt_utils.star(from=ref('int_shopify_gql__customer_email_rollup'), except=["orders_count", "total_spent"]) }}
+    from {{ ref('int_shopify_gql__customer_email_rollup') }}
+
+), orders as (
+
+    select *
+    from {{ ref('int_shopify_gql__emails__order_aggregates' )}}
+    where email is not null
+
+{% if var('shopify_using_abandoned_checkout', True) %}
```

Comment:
Shouldn't this be `shopify_gql_using_abandoned_checkout` instead?

Reply:
Confirmed that disabling `shopify_gql_using_abandoned_checkout` results in a compilation failure for this model.

Reply:
Ah yep, fixing